api/views.py

```diff
@@ -120,3 +120,11 @@ class AssetTypeViewSet(ModelViewSet):
     permission_classes = [IsAuthenticated, ]
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ['get', 'post']
+
+
+class AssetMakeViewSet(ModelViewSet):
+    serializer_class = AssetMakeSerializer
+    queryset = AssetMake.objects.all()
+    permission_classes = [IsAuthenticated, ]
+    authentication_classes = [FirebaseTokenAuthentication]
```

Comment:
Remember to add a trailing comma here like so `authentication_classes = [FirebaseTokenAuthentication, ]`